### PR TITLE
Update datapackage.json

### PR DESCRIPTION
**Pull request by richard-thonig**
added descriptions, will later ask Merce to review as well.

### CODE REVIEW

File: csp-guru/datapackage.json [hunk 1]
```diff
@@ -1,11 +1,161 @@
 {
-    "title": "A database of concentrated solar power plants of the world, for energy modellers and analysts.",
+    "title": "CSP.guru",
     "homepage": "https://csp.guru",
     "version": "2016-08-30",
+    "description": "A database of concentrated solar power plants of the world, for energy modellers and analysts.",
```
Comment (timtroendle) [inline csp-guru/datapackage.json]: I removed the comma everywhere else and I'd remove it here as well.

File: csp-guru/datapackage.json [hunk 2]
```diff
@@ -1,11 +1,161 @@
...
     "resources": [
         {
-            "path": "csp-guru.csv",
-            "hash": "0ed89d2fe7c96615ad70d79ce64693a8"
+            "path": "CSP.guru-2016-08-30.csv",
```
Comment (timtroendle) [inline csp-guru/datapackage.json]: The path is wrong (it was correct before you changed it). This also leads to the failed checks.

File: csp-guru/datapackage.json [hunk 3]
```diff
@@ -1,11 +1,161 @@
...
+                        "name": "Title",
+                        "description": "The name of the CSP plant"
+                    },
+                    {
+                        "name": "Year_operational",
+                        "description": "indicates the year of the first power output"
+                    },
+                    {
+                        "name": "Status",
+                        "description": "Dinstinguishes if a CSP plant is operational or under construction"
```
Comment (timtroendle) [inline csp-guru/datapackage.json]: Typo in "distinguishes"

File: csp-guru/datapackage.json [hunk 4]
```diff
@@ -1,11 +1,161 @@
...
+                        "name": "Storage_capacity",
+                        "description": " how many hours the turbine can run on stored heat alone"
+                    },
+                    {
+                        "name": "Field_size",
+                        "description": "the mirror area in square meters"
+                    },
+                    {
+                        "name": "Cooling",
+                        "description": "Cooling technology: dry, wet ,or hybrid"
```
Comment (timtroendle) [inline csp-guru/datapackage.json]: There is something wrong with the comma.

File: csp-guru/datapackage.json [hunk 5]
```diff
@@ -1,11 +1,161 @@
...
+                        "name": "Capacity",
+                        "description": "Nominal capacity of the turbine [MW]"
+                    },
+                    {
+                        "name": "Storage_capacity",
+                        "description": " how many hours the turbine can run on stored heat alone"
+                    },
+                    {
+                        "name": "Field_size",
+                        "description": "the mirror area in square meters"
```
Comment (timtroendle) [inline csp-guru/datapackage.json]: I'd be consistent with the formatting of units. As you used "[unit]" in most or all other cases, I'd use it here as well: [m^2].

File: csp-guru/datapackage.json [hunk 6]
```diff
@@ -1,11 +1,161 @@
...
+                        "name": "Support_duration",
+                        "description": "duration of PPA, FiT, etc."
+                    },
+                    {
+                        "name": "Enipedia_ID",
+                        "description": "https://enipedia.org/wiki/Enipedia"
+                    },
+                    {
+                        "name": "SolarPACES_ID",
+                        "description": "dasdas"
```
Comment (timtroendle) [inline csp-guru/datapackage.json]: No.

File: csp-guru/datapackage.json [hunk 7]
```diff
@@ -19,5 +169,11 @@
             "email": "merce.larbordena@usys.ethz.ch",
             "role": "author"
         }
-    ]
+    ],
+    "licenses":[
+      {
+  "name": "CC BY-NC 4.0",
```
Comment (timtroendle) [inline csp-guru/datapackage.json]: Something is completely wrong with the indentation here. Make sure you use 4 spaces per indentation level as this was used in the rest of the file and then bring it on the correct indentation level. An editor like Atom can do that automatically for you if you don't want to do it manually.

File: csp-guru/datapackage.json [hunk 8]
```diff
@@ -1,11 +1,161 @@
...
         {
-            "path": "csp-guru.csv",
-            "hash": "0ed89d2fe7c96615ad70d79ce64693a8"
+            "path": "CSP.guru-2016-08-30.csv",
+            "hash": "0ed89d2fe7c96615ad70d79ce64693a8",
+            "schema": {
+                "fields": [
+                    {
+                        "name": "Title",
+                        "description": "The name of the CSP plant"
```
Comment (timtroendle) [inline csp-guru/datapackage.json]: I'd be consistent with your format: either always start the description with a capital letter or never. A capital letter start would need a full stop at the end I'd say.

File: csp-guru/datapackage.json [hunk 9]
```diff
@@ -1,11 +1,161 @@
...
     "resources": [
         {
-            "path": "csp-guru.csv",
-            "hash": "0ed89d2fe7c96615ad70d79ce64693a8"
+            "path": "CSP.guru-2016-08-30.csv",
+            "hash": "0ed89d2fe7c96615ad70d79ce64693a8",
+            "schema": {
```
Comment (timtroendle) [inline csp-guru/datapackage.json]: Currently this throws warnings next to the error. I believe we should make this whole thing a Tabular Data Package. We would need to add `"profile": "tabular-data-package"` on the first level and `"profile": "tabular-data-resource"` on the level of the resource. 

There is an example here:
http://frictionlessdata.io/specs/tabular-data-package/#specification

File: csp-guru/datapackage.json [hunk 10]
```diff
@@ -1,11 +1,161 @@
...
+                        "name": "Technology",
+                        "description": "Dinstinguishes CSP designs in Trough, Tower, Hybrid, Fresnel "
+                    },
+                    {
+                        "name": "Capacity",
+                        "description": "Nominal capacity of the turbine [MW]"
+                    },
+                    {
+                        "name": "Storage_capacity",
+                        "description": " how many hours the turbine can run on stored heat alone"
```
Comment (timtroendle) [inline csp-guru/datapackage.json]: I'd add `[h]`.

File: csp-guru/datapackage.json [hunk 11]
```diff
@@ -1,11 +1,161 @@
...
+                        "name": "Remuneration_2015USD",
+                        "description": "indicates the level of FiT or PPA [ct/kWh], 2015 USD "
+                    },
+                    {
+                        "name": "Remuneration_2015EUR",
+                        "description": "indicates the level of FiT or PPA in [ct/kWh], 2015 EURO "
+                    },
+                    {
+                        "name": "LCOE_5_2015USD",
+                        "description": "Levelized Cost of Electricity with 5 years payback [ct/kWh], 2015 USD discounted"
```
Comment (timtroendle) [inline csp-guru/datapackage.json]: This is not sufficient to understand where the number comes from. But that should not be part of this change I guess.

File: csp-guru/datapackage.json [hunk 12]
```diff
@@ -1,11 +1,161 @@
...
+                        "name": "Other_support",
+                        "description": "Financial assistence to project developer like tax credits, cheap credits, etc."
+                    },
+                    {
+                        "name": "Owner",
+                        "description": "Owner of the CSP plant"
+                    },
+                    {
+                        "name": "Support_duration",
+                        "description": "duration of PPA, FiT, etc."
```
Comment (timtroendle) [inline csp-guru/datapackage.json]: `[a]` ?

File: csp-guru/datapackage.json (hunk 9 above)
Comment (timtroendle) [reply csp-guru/datapackage.json]: I just tried it and no, this does not remove the validation warning. I'd make this a tabular data package anyways.

File: csp-guru/datapackage.json (hunk 2 above)
Comment (richard-thonig) [reply csp-guru/datapackage.json]: I did not change it - I think I used an old template.
csp-guru.csv is correct, right?

Comment (timtroendle) [reply csp-guru/datapackage.json]: Jup.

File: csp-guru/datapackage.json [hunk 13]
```diff
@@ -1,15 +1,167 @@
...
+                    {
+                        "name": "Load_factor",
+                        "description": "percentage of maximum yearly power output, increases with storage"
+                    },
+                    {
+                        "name": "Total_cost",
+                        "description": "in thousand local currency"
+                    },
+                    {
+                        "name": "Specific_investment_cost_2015USD",
```
Comment (mercelabordena) [inline csp-guru/datapackage.json]: "name": "Investment_cost_2015USD",

File: csp-guru/datapackage.json [hunk 14]
```diff
@@ -1,15 +1,167 @@
...
+                        "description": "investment cost per MW [2015 USD], discounted"
+                    },
+                    {
+                        "name": "Specific_investment_cost_2015EUR",
```
Comment (mercelabordena) [inline csp-guru/datapackage.json]: "name": "Investment_cost_2015EUR",

File: csp-guru/datapackage.json [hunk 15]
```diff
@@ -1,15 +1,167 @@
...
+                    {
+                        "name": "HCE_manufacturer",
+                        "description": "heat collecting element (HCE) manufacturer"
+                    },
+                    {
+                        "name": "SCA_manufacturer",
+                        "description": "solar collector assemblies (SCA) manufacturer"
+                    },
+                    {
+                        "name": "Financed_through",
```
Comment (mercelabordena) [inline csp-guru/datapackage.json]: "name": "Financing_mechanism",

File: csp-guru/datapackage.json [hunk 16]
```diff
@@ -1,15 +1,167 @@
...
+                        "name": "Remuneration_2015USD",
+                        "description": "indicates the level of FiT or PPA [ct/kWh], 2015 USD "
+                    },
+                    {
+                        "name": "Remuneration_2015EUR",
+                        "description": "indicates the level of FiT or PPA in [ct/kWh], 2015 EURO "
+                    },
+                    {
+                        "name": "LCOE_5_2015USD",
+                        "description": "levelized cost of electricity with 5 years payback [ct/kWh], 2015 USD discounted"
```
Comment (mercelabordena) [inline csp-guru/datapackage.json]: "description": "levelized cost of electricity with 5% weighted average cost of capital [ct/kWh], 2015 USD discounted"

In the csp.guru wiki says wacc (weighted average cost of capital), I would use that or discount rate, or if you really want to write "payback" at least "discounted payback period". This change would apply to the other three below.

File: csp-guru/datapackage.json [hunk 17]
```diff
@@ -1,15 +1,167 @@
...
+                        "name": "LCOE_5_2015EUR",
+                        "description": "levelized cost of electricity with 5 years payback [ct/kWh], 2015 EURO discounted"
+                    },
+                    {
+                        "name": "LCOE_10_2015EUR",
+                        "description": "levelized cost of electricity with 10 years payback [ct/kWh], 2015 EURO discounted"
+                    },
+                    {
+                        "name": "Other_support",
+                        "description": "financial assistence to project developer like tax credits, cheap credits, etc."
```
Comment (mercelabordena) [inline csp-guru/datapackage.json]: "description": "financial assistance to project developer: tax credits, cheap credits, etc."

File: csp-guru/datapackage.json [hunk 18]
```diff
@@ -1,15 +1,167 @@
...
+                    },
+                    {
+                        "name": "Owner",
+                        "description": "owner of the CSP plant"
+                    },
+                    {
+                        "name": "Support_duration",
+                        "description": "duration of PPA, FiT, etc. [a]"
```
Comment (mercelabordena) [inline csp-guru/datapackage.json]: "description": "duration of PPA, FiT, etc. [yr]"

Above you used once [y] and another [a]. Suggest to use always [yr]

File: csp-guru/datapackage.json [hunk 19]
```diff
@@ -1,15 +1,167 @@
...
+                        "name": "Enipedia_ID",
+                        "description": "https://enipedia.org/wiki/Enipedia"
+                    },
+                    {
+                        "name": "SolarPACES_ID",
+                        "description": "SolarPACES_ID "
+                    },
+                    {
+                        "name": "Sources",
+                        "description": "indicates were the information for each plant was taken from"
```
Comment (mercelabordena) [inline csp-guru/datapackage.json]: "description": "indicates where the information for each plant was taken from"

File: csp-guru/datapackage.json (hunk 15 above)
Comment (timtroendle) [reply csp-guru/datapackage.json]: Sound better and in the future we can change that but for the moment we need to stick to the published dataset and that says `Financed_through `.

File: csp-guru/datapackage.json [hunk 20]
```diff
@@ -19,5 +171,12 @@
             "email": "merce.larbordena@usys.ethz.ch",
```
[the remark this answers is not in the document]
Comment (timtroendle) [reply csp-guru/datapackage.json]: Sorry about that. 😬

File: csp-guru/datapackage.json [hunk 21]
```diff
@@ -1,15 +1,167 @@
...
+                    {
+                        "name": "Field_size",
+                        "description": "the mirror area [㎡]"
+                    },
+                    {
+                        "name": "Cooling",
+                        "description": "cooling technology: dry, wet, or hybrid"
+                    },
+                    {
+                        "name": "Expected_generation",
```
[the remark this answers is not in the document]
Comment (timtroendle) [reply csp-guru/datapackage.json]: As above: good suggestion but will need to wait for next version.

File: csp-guru/datapackage.json [hunk 22]
```diff
@@ -1,15 +1,167 @@
...
+                        "name": "Sources",
+                        "description": "indicates were the information for each plant was taken from"
+                    }
+                ]
+            }
         }
     ],
     "contributors": [
-        {
+      {
```
Comment (timtroendle) [inline csp-guru/datapackage.json]: This change is wrong.

File: csp-guru/datapackage.json (hunk 16 above)
Comment (richard-thonig) [reply csp-guru/datapackage.json]: thanks for pointing out that it's 5% not 5 years...

File: csp-guru/datapackage.json (hunk 18 above)
Comment (richard-thonig) [reply csp-guru/datapackage.json]: agreed.

File: csp-guru/datapackage.json (hunk 22 above)
Comment (mercelabordena) [reply csp-guru/datapackage.json]: Hey, yes, I don't even know what's that.

**Comment by richard-thonig** [comment]
Ok. I have integrated Merces feedback
